### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by kozi** [mozilla/pdf.js#19150]
### Attach (recommended) or Link to PDF file

The function `process.getBuiltinModule();` requires  **Node v20.16.0**

https://nodejs.org/en/blog/release/v20.16.0#2024-07-24-version-20160-iron-lts-marco-ippolito

https://github.com/mozilla/pdf.js/pull/18959/files#diff-23805277aa87af178b8804b8c8f46f9655d0cc44a10e1a32a0d3ebc12ed7ec0eR31 

### Web browser and its version

Node v2.15.1 (Alpine Linux v3.20)

### Operating system and its version

Node v2.15.1 (Alpine Linux v3.20)

### PDF.js version

v4.9.124

### Is the bug present in the latest PDF.js version?

Yes

### Is a browser extension

No

### Steps to reproduce the problem

Try to use the with Node v2.15.1

### What is the expected behavior?

No error

### What went wrong?

Function not defined error.

**Comment by nicolo-ribaudo** [comment]
v20 is the minimum supported version, see https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#legacy-build.

**Comment by Snuffleupagus** [comment]
> The function `process.getBuiltinModule();` requires **Node v20.16.0**

Yes, this is expected and you'll thus need to update the Node.js version.

**Comment by kozi** [comment]
> > The function `process.getBuiltinModule();` requires **Node v20.16.0**
> 
> Yes, this is expected and you'll thus need to update the Node.js version.

Node **v20.15.1** is **~v20**
